examples/spring-boot-minimal-kotlin/src/test/kotlin/example/springframework/boot/minimal/kotlin/HelloApplicationIntegrationTest.kt

```diff
@@ -0,0 +1,35 @@
+package example.springframework.boot.minimal.kotlin
+
+import com.linecorp.armeria.client.WebClient
+import com.linecorp.armeria.common.HttpStatus
+import com.linecorp.armeria.server.Server
+import net.javacrumbs.jsonunit.fluent.JsonFluentAssert
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.ActiveProfiles
+
+@ActiveProfiles("testbed")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE)
+class HelloApplicationIntegrationTest (@Autowired server: Server) {
+
+    private val client: WebClient = WebClient.of("http://localhost:" + server.activeLocalPort())
+
+    @Test
+    fun success() {
+        val response = client.get("/hello/Spring").aggregate().join()
+        Assertions.assertThat(response.status()).isEqualTo(HttpStatus.OK)
+        Assertions.assertThat(response.contentUtf8())
+                  .isEqualTo("Hello, Spring! This message is from Armeria annotated service!")
+    }
+
+    @Test
+    fun failure() {
+        val response = client.get("/hello/a").aggregate().join()
+        Assertions.assertThat(response.status()).isEqualTo(HttpStatus.BAD_REQUEST)
+        JsonFluentAssert.assertThatJson(response.contentUtf8())
+                        .node("message")
+                        .isEqualTo("hello.name: name should have between 3 and 10 characters")
```

Comment:
No static imports in Kotlin? What's the most popular assertion library in Kotlin ecosystem? Should we keep using AssertJ?

Reply:
> No static imports in Kotlin?
Can be used.
> What's the most popular assertion library in Kotlin ecosystem?
It seems that [the official test framework](https://github.com/kotlintest/kotlintest) contains assertions.
> Should we keep using AssertJ?
Most Kotlin assertion libraries have a library of assertj wrappers.